Competition2017/src/competition/subsystems/autonomous/DriveToBoilerWithVisionCommandGroup.java

```diff
@@ -0,0 +1,53 @@
+package competition.subsystems.autonomous;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+
+import competition.subsystems.drive.commands.CalculateDistanceOfParallelCommand;
+import competition.subsystems.drive.commands.DriveForDistanceCommand;
+import competition.subsystems.drive.commands.DriveInfinitelyCommand;
+import competition.subsystems.drive.commands.RotateToHeadingCommand;
+import competition.subsystems.vision.commands.RotateRobotToBoilerCommand;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class DriveToBoilerWithVisionCommandGroup extends CommandGroup{
```

Comment:
Since we're growing a collection of "drive to" command groups it might be handy to have a comment up top here discussing what situation this one is to be applied in.